tests/js/spec/stores/guideStore.spec.jsx

```diff
@@ -48,11 +49,33 @@ describe('GuideStore', function() {
     GuideStore.onFetchSucceeded(data);
     // GuideStore should prune steps that don't have anchors.
     expect(GuideStore.state.currentGuide.steps).toHaveLength(2);
+    expect(GuideStore.state.currentGuide.seen).toEqual(false);
     GuideStore.onNextStep();
     expect(GuideStore.state.currentStep).toEqual(1);
     GuideStore.onNextStep();
     expect(GuideStore.state.currentStep).toEqual(2);
     GuideStore.onCloseGuideOrSupport();
-    expect(GuideStore.state.guidesSeen).toEqual(new Set([1]));
+  });
+
+  it('should not show seen guides', function() {
+    data.issue.seen = true;
```

Comment:
data isn't being reset after each test, so once you do this it will be seen for all the tests. to prevent this, set `data` in `beforeEach` so it will be reset each time (and replace the top level declaration with simply `let data;`)

Reply:
fixed